js/src/spreadsheet.widget.js

```diff
@@ -576,14 +576,27 @@ class ModinSpreadsheetView extends widgets.DOMWidgetView {
     }
 
     this.slick_grid.onCellChange.subscribe((e, args) => {
-      var column = this.columns[args.cell].name;
+      var column = args.grid.getColumns()[args.cell].name;
       var data_item = this.slick_grid.getDataItem(args.row);
       var msg = {'row_index': data_item.row_index, 'column': column,
                  'unfiltered_index': data_item[this.index_col_name],
                  'value': args.item[column], 'type': 'edit_cell'};
       this.send(msg);
     });
 
+    this.slick_grid.onColumnsReordered.subscribe((e, args) => {
+      var column_names = [];
+      var num_columns = args["grid"]["getColumns"]().length;
```

Comment:
To follow previous convention, use . instead of [] for accessing members
```suggestion
      var num_columns = args.grid.getColumns().length;
```